### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by ticotheps**
Hey @jmaxt12 ,

Here is my submission for Sprint Challenge #8: Redux-Smurfs!

Let's GET IT,
Teeks "McGeeks" Thepsourinthone

### CODE REVIEW

**Comment by RaDeleon** [comment]
Good job getting this done on the second attempt also 90% of the students didn't bother to style it so 🥇 for that. Not much else to say. Nice work! 👍

**Comment by ticotheps** [comment]
Thanks for taking the time to grade it, Angelo!

Have a great weekend!
-Tico

File: Answers.md [hunk 1]
```diff
@@ -1,6 +1,121 @@
...
+    club, to get in, (2) Jake moves slowly toward the entrance, which is 
+    guarded by a ‘bouncer’ or security guard, (3) when Jake reaches the 
+    entrance, a bouncer will inspect him (and his ID) and then make one of 
+    two decisions: (a) let him into the club or (b) deny him access. If we 
+    think of a reducer as a “night club” and Jake as the “action” that’s 
+    trying to get into the “REDUCERS Night Club” (a popular place because 
+    drink prices are often ‘reduced’ - haha), then we can think of the 
+    middleware as that “bouncer”! Basically, the job of a middleware is to 
+    intercept actions before they reach the reducers.
+
```
Comment (jmaxt12) [inline Answers.md]: hahahaha This is so great! Great analogy!

File: Answers.md [hunk 2]
```diff
@@ -1,6 +1,121 @@
...
+    by another function) is returned INSTEAD of an action (which is just an 
+    object) when redux-thunk intercepts that action creator, redux-thunk will 
+    invoke that function and pass in the dispatch function as an argument to 
+    it (AKA the bouncer finds out that Jake is ACTUALLY related to Justin 
+    Timberlake, who will be playing at REDUCERS that night, so, in hopes of 
+    a possible raise, the bouncer calls management to tell them to prepare 
+    a VIP table with the finest bottle of Ciroc for Jake before granting 
+    him access to the night club so he can enjoy the VIP table and drinks in
+    addition to the dancing).
+
```
Comment (jmaxt12) [inline Answers.md]: Oh man these answers are so good and thorough. I'm glad I got to spend to time to read them. Great work and understanding on this!

**Pull request merged**